doc/schemas/listpeers.schema.json

```diff
@@ -165,1002 +164,1025 @@
-                    "enum": [
-                      "option_static_remotekey",
-                      "option_anchor_outputs",
-                      "option_zeroconf"
-                    ],
-                    "description": "BOLT #9 features which apply to this channel"
-                  }
-                },
-                "funding": {
-                  "type": "object",
-                  "additionalProperties": false,
-                  "required": [
-                    "local_funds_msat",
-                    "remote_funds_msat"
-                  ],
-                  "properties": {
-                    "local_msat": {
-                      "type": "msat",
-                      "description": "Amount of channel we funded (deprecated)"
-                    },
-                    "remote_msat": {
-                      "type": "msat",
-                      "description": "Amount of channel they funded (deprecated)"
-                    },
-                    "pushed_msat": {
-                      "type": "msat",
-                      "description": "Amount pushed from opener to peer"
-                    },
-                    "local_funds_msat": {
-                      "type": "msat",
-                      "description": "Amount of channel we funded"
-                    },
-                    "remote_funds_msat": {
-                      "type": "msat",
-                      "description": "Amount of channel they funded"
-                    },
-                    "fee_paid_msat": {
-                      "type": "msat",
-                      "description": "Amount we paid peer at open"
-                    },
-                    "fee_rcvd_msat": {
-                      "type": "msat",
-                      "description": "Amount we were paid by peer at open"
-                    }
-                  }
-                },
-                "to_us_msat": {
-                  "type": "msat",
-                  "description": "how much of channel is owed to us"
-                },
-                "min_to_us_msat": {
-                  "type": "msat",
-                  "description": "least amount owed to us ever"
-                },
-                "max_to_us_msat": {
-                  "type": "msat",
-                  "description": "most amount owed to us ever"
-                },
-                "total_msat": {
-                  "type": "msat",
-                  "description": "total amount in the channel"
-                },
-                "fee_base_msat": {
-                  "type": "msat",
-                  "description": "amount we charge to use the channel"
-                },
-                "fee_proportional_millionths": {
+                  "type": "hex",
+                  "description": "bitmap of BOLT #9 features from peer's INIT message"
+                }
+              }
+            }
+          }
+        ]
+      },
+      "channels": {
+        "deprecated": "true",
```

Comment:
Ah, we don't normally put deprecated options in the schema at all (if we want to test them, we disable schema checking).

OK that you did though!

Reply:
Yeah, but this was requested here https://github.com/ElementsProject/lightning/pull/5022#discussion_r949215155

We should add some policy regarding the auto generated files?